databricks/koalas/series.py

```diff
@@ -929,6 +929,75 @@ def sort_values(self, ascending: bool = True, inplace: bool = False,
         else:
             return ks_
 
+    def sort_index(self, axis: int = 0, level: int = None, ascending: bool = True,
+                   inplace: bool = False, kind: str = None, na_position: str = 'last') \
+            -> Optional['Series']:
+        """
```

Comment:
The docstring is almost the same as `DataFrame.sort_index()`. You can use 
```python
sort_index.__doc__ = DataFrame.sort_index.__doc__
```

Reply:
Fair point but how does this deal with the doctests that need to use Series instead of DataFrames?

Reply:
You can include one example of Series in the same doctest of DataFrame.